doc/api-extensions.md

```diff
@@ -2407,3 +2407,8 @@ This extension adds a new image restriction, `requirements.nesting` which when `
 
 Adds the {config:option}`instance-miscellaneous:linux.kernel_modules.load` container configuration option. If the option is set to `ondemand`, the `finit_modules()` syscall is intercepted and a privileged user in the container's user namespace can load the Linux kernel modules specified in the
 allow list {config:option}`instance-miscellaneous:linux.kernel_modules`.
+
```

Comment:
When picking an API extension name, its worth looking at previous ones to try and keep to the naming pattern/theme.

The relevant ones I can find for this are:

`network_allocations`
`network_forward`
`network_load_balancer`

So we can say that this new API extension should start with `network_` at a minimum.

So how about: `network_allocate_external_ips`?